tests/hugetlb.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 And Group
+//
+// SPDX-License-Identifier: Apache-2.0 or MIT
+//
+
+//! Integration tests about the hugetlb subsystem
+use cgroups::hugetlb::HugeTlbController;
+use cgroups::{Cgroup, Hierarchy};
+use cgroups::Controller;
+
+use cgroups::error::ErrorKind::*;
+use cgroups::error::*;
+
+#[test]
+fn test_hugetlb_sizes() {
+    // now only v2
+    if cgroups::hierarchies::is_cgroup2_unified_mode() {
+        return
+    }
+
+    let h = cgroups::hierarchies::auto();
+    let h = Box::new(&*h);
+    let cg = Cgroup::new(h, String::from("test_hugetlb_sizes"));
+    {
+        let hugetlb_controller: &HugeTlbController = cg.controller_of().unwrap();
+        let sizes = hugetlb_controller.get_sizes();
+
+        let size = "2MB";
```

Comment:
hugetlb might be either `2MB` or `1GB`. Maybe test it with 
```
        assert_eq!(
            sizes.contains(&"2MB".to_string()) || size.contains(&"1GB".to_string()),
            true
        );
```

Reply:
I think this may be depend on test environment, so just to avoid the extract size by hard coding.